OpenOversight/app/utils/general.py

```diff
@@ -159,6 +159,11 @@ def str_is_true(str_) -> bool:
     return bool(strtobool(str_.lower()))
 
 
+def url_for_target_null_values_skipped(target, **kwargs):
```

Comment:
Could you put types (parameters and returns) on all of these functions?

Reply:
Done